src/app/app.config.ts

```diff
@@ -29,7 +27,7 @@ export class AppConfig {
           }, error => {
             console.error('Configuration file "config.json" could not be read');
             reject();
-            return Observable.throw(error.json().error || 'Server error');
+            return throwError(error.json().error || 'Server error');
```

Comment:
this is a weird little bit of code - why is reject called and an RX.js error returned?

I would have just expected `error => reject`

Reply:
I just replaced a deprecated method here.
rxjs just provides observable/promise operators.
not really sure that we really need to throw an error, but I didn't actually write this, so don't know the full logic